x-pack/plugins/enterprise_search/public/applications/applications/components/playground/playground.tsx

```diff
@@ -13,8 +13,8 @@ import { useValues } from 'kea';
 
 import { i18n } from '@kbn/i18n';
 
+import { EnterpriseSearchContentPageTemplate } from '../../../enterprise_search_content/components/layout/page_template';
```

Comment:
I think we'll want to use `EnterpriseSearchApplicationsPageTemplate` here instead.

Reply:
updated